hedera-node/src/main/java/com/hedera/services/state/submerkle/EntityId.java

```diff
@@ -91,6 +91,17 @@ public static EntityId fromIdentityCode(int code) {
 		return new EntityId(DEFAULT_SHARD, DEFAULT_REALM, numFromCode(code));
 	}
 
+	/**
+	 * Builds an entity id from just the account number.
+	 *
+	 * @param accountNum
+	 * 		the account number of the entity id.
+	 * @return the equivalent entity id using the default shard and realm.
+	 */
+	public static EntityId fromAccountNum(long accountNum) {
```

Comment:
It's slightly preferable to make this as a factory on [the `STATIC_PROPERTIES` singleton](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/src/main/java/com/hedera/services/context/properties/StaticPropertiesHolder.java#L72), since that component gets the "true" shard and realm from [here](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/src/main/resources/bootstrap.properties#L36).

But there is obviously no risk of this method becoming incorrect suddenly, or soon. 😁

Reply:
I will keep EntityId static factory method with EntityId, but will make use of STATIC_PROPERTIES singleton in the class.